gems/manageiq-providers-amazon/app/models/manageiq/providers/amazon/cloud_manager/vm/operations.rb

```diff
@@ -1,5 +1,6 @@
 module ManageIQ::Providers::Amazon::CloudManager::Vm::Operations
   include_concern 'Guest'
+  include_concern 'Instance'
```

Comment:
`Instance` feels strange to me. How about renaming it to `Availabilty`, which is somehow off too, but at least consistent? 
Or, maybe even better, move the `validate_timeline` method to `CloudManager::Vm` directly